src/Model/Svg/SvgFile.php

```diff
@@ -430,6 +430,38 @@ public function getInFileTranslations(): array
         return $this->inFileTranslations;
     }
 
+    /**
+     * Set translations for a single language.
+     * @param string $lang Language code of the translations being provided.
+     * @param string[] $translations Array of tspan-IDs to message texts.
+     * @return string[][]
+     */
+    public function setTranslations(string $lang, array $translations): array
```

Comment:
Hm, this is a bit confusing, but it's also an English preference, so I leave that to proper English speakers -- but... "set translation**s**" sound to me like you can set multiple translations to a single file, and to me, "translations" are per language, so multiple translations = multiple languages.

If this is only me, leave as is, but if this is confusing, then you can change it to either 'setTranslation' (singular) that acceps language + key/value map OR set it to 'setTranslationSets' or something. Translation is usually a grouping of langA->langB words for a single language, isn't it?